common/reviews/api/rush-stack-compiler-2.4.api.md

```diff
@@ -69,35 +69,35 @@ export abstract class RushStackCompilerBase<TOptions extends IRushStackCompilerB
 export class StandardBuildFolders {
     constructor(projectFolderPath: string);
     // (undocumented)
-    readonly distFolderPath: string;
+    get distFolderPath(): string;
     // (undocumented)
-    readonly libFolderPath: string;
+    get libFolderPath(): string;
```

Comment:
When `readonly`->`get`, that indicates that you upgraded to TypeScript 3.7 and are breaking library compatibility with all older compilers. Was that intentional?

Reply:
Yeah it was intentional. We were on 3.5, which is pretty old at this point.

Reply:
Undid this change.